packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts

```diff
@@ -86,3 +89,30 @@ async function getCollectionMetadata(
   const metadataResponse = await fetch(metadata.collectionUri)
   return await metadataResponse.json()
 }
+
+async function getNftMetadataz(nftIds: string[], explorerProvider: ExplorerProvider) {
```

Comment:
Does NFT metadata fetch use backoff retry when the query fails?

Reply:
Yes, it uses SWR's default exponential backoff retry strategy when query fails.

Reply:
Yes. Should be consistent